File: Transport/JS/Transport/view/battery/BatteryType.js
T.view.BatteryType = Ext.extend(Kdn.view.BaseGrid, {
    modelName: 'BatteryType',
    constructor: function(cfg) {
        cfg = cfg || {};
        Ext.apply(cfg, {

            colModel: new Ext.grid.ColumnModel({
               defaults:{filter:{}},
                columns: [
                    {
                        dataIndex: 'BatteryTypeId',
                        header: 'Код',
                        width: 130,
                        editor: { xtype: 'kdn.editor.id' }
                    },
                    {
                        dataIndex: 'BatteryTypeName',
                        header: 'Наименование типа АКБ',
                        width: 450,
                        editor: { xtype: 'kdn.editor.textfield' }
                    },
                    {
                        dataIndex: 'Warrantly',
                        header: 'Гарантия, мес.',
                        width: 130,
                        editor: { xtype: 'kdn.editor.numberfield' }
                    },
                    {
                        dataIndex: 'KmNorm',
                        header: 'Норма на пробег, км',
                        width: 150,
                        editor: { xtype: 'kdn.editor.textfield' }
                    },
                    {
                        dataIndex: 'MhNorm',
                        header: 'Норма на маш.часы',
                        width: 150,
                        editor: { xtype: 'kdn.editor.textfield' }
                    }
                ]
            })
        });

        T.view.BatteryType.superclass.constructor.call(this, cfg);

    }
});

Ext.reg('view.batterytype', T.view.BatteryType);

File: Transport/Models/battery/BatteryType.cs
using Castle.ActiveRecord;
using Kdn.Ext.Attributes;

namespace Transport.Models
{
   [ActiveRecord, Model]
   public class BatteryType : ActiveRecordBase<BatteryType>
   {
      [PrimaryKey, IdProperty]
      public int BatteryTypeId { get; set; }

      [Property]
      public string BatteryTypeName { get; set; }

      //Гарантия мес.
      [Property]
      public short? Warrantly { get; set; }

      //Норма на пробег
      [Property]
      public int? KmNorm { get; set; }

      //Норма на моточасы
      [Property]
      public int? MhNorm { get; set; }
   }
}
